errdefs/grpc_test.go

```diff
@@ -85,9 +85,6 @@ func TestGRPCRoundTrip(t *testing.T) {
 			ferr := FromGRPC(gerr)
 			t.Logf("recovered: %v", ferr)
 
-			if errors.Cause(ferr) != testcase.cause {
```

Comment:
Use Unwrapf instead of removing?

Reply:
Actually the following assertion use `errors.Is` which calls `errors.Unwrap` inside and covered this case.